src/prune.ts

```diff
@@ -0,0 +1,186 @@
+import { join, sep as pathSeparator } from 'path'
+import Promise = require('any-promise')
+import { EventEmitter } from 'events'
+import { Emitter } from './interfaces'
+import { findProject } from './utils/find'
+import { readConfig, isFile, transformDtsFile, rimraf } from './utils/fs'
+import { CONFIG_FILE, TYPINGS_DIR, DTS_BROWSER_FILE, DTS_MAIN_FILE } from './utils/config'
+import { relativeTo, isDefinition } from './utils/path'
+
+const AMBIENT_TYPE_DIR = 'ambient'
```

Comment:
This is repeated from here: https://github.com/typings/core/blob/master/src/utils/path.ts#L8. This could be pulled into a config file if desired.